atlp-rwanda/hackers-ec-Fe
# #187419007 Implementing Seller Authentication and Two Factor Authentication

## What does this PR do?

Implements Seller Authentication and Two Factor Authentication 

## Description of Task to be completed? 

Enable the seller to access the app using the **OTP code** sent via email after logging in

## How should this be manually tested?

Open our app try to login with seller credentials, find **OTP code** on the email you used logging in and **click** on the link there to go to the 2FA page to insert the code and click verify button to access the app.

## What are the relevant pivotal tracker/Trello stories?

#187419007
